handlers/handler.go

```diff
@@ -78,14 +80,21 @@ func IndexS3Object(s3objectURL string) {
 	//     <dataguid>/<uuid>/<filename>
 	//
 	// we want to keep the `<dataguid>/<uuid>` part
+	key = strings.Trim(key, "/")
```

Comment:
> When a user uploads files using gen3-client with "--include-subdirname" option

I don't see where you check if the user did or did not use the option

Reply:
As far as I know, from this job, there is no way to tell whether or not the user supplied `--include-subdirname` to `gen3-client` while uploading, so the presence or absence of the option wouldn't be explicitly checked here.

Reply:
oh right gotcha